prov/efa/src/rxr/rxr_read.h

```diff
@@ -97,5 +97,16 @@ void rxr_read_handle_read_completion(struct rxr_ep *ep, struct rxr_pkt_entry *pk
 
 int rxr_read_handle_error(struct rxr_ep *ep, struct rxr_read_entry *read_entry, int ret);
 
+struct rxr_local_read_entry {
```

Comment:
This should start with `enum rxr_x_entry_type type;` and a new type needs to be added if you're going to set a pointer to this in `pkt_entry->x_entry`.

I also didn't go follow this code yet but if this packet is passed to `rxr_cq_handle_cq_error` it also needs to be updated to handle this new message metadata structure.

Reply:
added ```RXR_LOCAL_ENTRY_TYPE``` to ```enum rxr_x_entry_type```, also added a section in ```rxr_cq_handle_cq_error```.